static/styles/right-sidebar.scss

```diff
@@ -130,6 +141,7 @@
 
 .user_sidebar_entry .count,
 .group-pms-sidebar-entry .count {
+    float: right;
```

Comment:
Is the `float: right` property necessary? Doesn't look like there's a difference if I comment it out